next.config.js

```diff
@@ -10,8 +10,8 @@ module.exports = {
     ];
   },
   webpack(config) {
+    config.resolve.alias['appConfig'] = path.join(__dirname, 'src/appConfig');
```

Comment:
Rename to `app`?

Reply:
Gunna hold off. You were suggesting this because in your PR you create an `app` dir. But I don't want to touch that in this one.